app/jobs/whats_app_adapter/three_sixty_dialog/create_welcome_message_template_job.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'net/http'
+
+module WhatsAppAdapter
+  module ThreeSixtyDialog
+    class CreateWelcomeMessageTemplateJob < ApplicationJob
```

Comment:
How do we trigger this and where does `template_text` come from?

Do we trigger everything manually from `bin/rails c`?

Reply:
Yes, it just saves me time from not needing to copy paste the net http commands. I was thinking of the template name and how it makes sense to just copy paste the way we  capture the text in the welcome message in the send_welcome_message!